docs/agents/sematext-agent/packages/data.md

```diff
@@ -0,0 +1,13 @@
+title: Package Data
+description: Package Data collected by Sematext Agent
+
+Package payloads are comprised of various fields as summarized in the table. Sematext Agent currently supports Python, Node.js and Debian-based packages.
```

Comment:
Don't we also collect info about RPMs?

Reply:
We don't, because of obstacles we hit we tried to retrieve RPM packages from containers. I remember you recently posted a link to the github repo that contains a Go package for reading RPMs directly from bdb. We can consider using that approach.

Reply:
OK.  We should add a ticket for that if we don't already have it.  I just checked JIRA and couldn't find it.